src/Tools/dotnet-user-jwts/src/Commands/CreateCommand.cs

```diff
@@ -0,0 +1,204 @@
+
+        var audience = audienceOption.HasValue() ? audienceOption.Value() : DevJwtCliHelpers.GetApplicationUrl(project);
+        if (audience is null)
+        {
+            Console.WriteLine("Could not determine the project's HTTPS URL. Please specify an audience for the JWT using the --audience option.");
+            isValid = false;
+        }
+        var issuer = issuerOption.HasValue() ? issuerOption.Value() : DevJwtsDefaults.Issuer;
+
+        var notBefore = DateTime.UtcNow;
+        if (notBeforeOption.HasValue())
+        {
+            if (!ParseDate(notBeforeOption.Value(), out notBefore))
+            {
+                Console.WriteLine(@"The date provided for --not-before could not be parsed. Ensure you use the format 'yyyy-MM-dd [[[[HH:mm]]:ss]]'.");
+                isValid = false;
+            }
+        }
+
+        var expiresOn = notBefore.AddMonths(6);
+        if (expiresOnOption.HasValue())
+        {
+            if (!ParseDate(expiresOnOption.Value(), out expiresOn))
+            {
+                Console.WriteLine(@"The date provided for -expires-on could not be parsed. Ensure you use the format 'yyyy-MM-dd [[[[HH:mm]]:ss]]'.");
+                isValid = false;
+            }
+        }
+
+        if (validForOption.HasValue())
+        {
+            if (!TimeSpan.TryParseExact(validForOption.Value(), _timeSpanFormats, CultureInfo.InvariantCulture, out var validForValue))
+            {
+                Console.WriteLine("The period provided for --valid-for could not be parsed. Ensure you use a format like '10d', '24h', etc.");
+            }
+            expiresOn = notBefore.Add(validForValue);
+        }
+
+        var roles = rolesOption.HasValue() ? rolesOption.Values : new List<string>();
+        var scopes = scopesOption.HasValue() ? scopesOption.Values : new List<string>();
+
+        var claims = new Dictionary<string, string>();
+        if (claimsOption.HasValue())
+        {
+            if (!DevJwtCliHelpers.TryParseClaims(claimsOption.Values, out claims))
+            {
+                Console.WriteLine("Malformed claims supplied. Ensure each claim is in the format \"name=value\".");
+                isValid = false;
+            }
+        }
+
+        return (new JwtCreatorOptions(scheme, name, audience, issuer, notBefore, expiresOn, roles, scopes, claims), isValid);
+
+        static bool ParseDate(string datetime, out DateTime parsedDateTime) =>
+            DateTime.TryParseExact(datetime, _dateTimeFormats, CultureInfo.InvariantCulture, DateTimeStyles.AssumeUniversal | DateTimeStyles.AdjustToUniversal, out parsedDateTime);
+    }
+
+    private static int Execute(
+        string projectPath,
+        JwtCreatorOptions options)
+    {
+        var project = DevJwtCliHelpers.GetProject(projectPath);
+        if (project == null)
+        {
+            Console.WriteLine($"No project found at `-p|--project` path or current directory.");
```

Comment:
```suggestion
            Console.WriteLine($"No project found at '{projectPath}' path or current directory.");
```

Reply:
`projectPath` is likely to be null if `GetProject` fails to return a valid project so using the argument name to safeguard against printing nulls.